src/App.vue

```diff
@@ -700,13 +701,16 @@ export default {
       // Set parties
       this.storeParties({ data: { parties: application.parties || [] } })
 
+      const description = GetCorpFullDescription(this.getLegalType)
+      const name = EnumUtilities.filingTypeToName(header.name)
+
       // add this as a filing (for Filing History List)
       const filingItem = {
         availableOnPaperOnly: header.availableOnPaperOnly,
         businessIdentifier: this.getIdentifier,
         commentsCount: filedApplication.commentsCount,
         commentsLink: filedApplication.commentsLink,
-        displayName: EnumUtilities.filingTypeToName(header.name),
+        displayName: `${description} ${name}`,
```

Comment:
Collateral fix: completed application didn't match filing display name on business dashboard.

Application dashboard (note in URL that id=Txxx):
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/00521e3c-9f4c-4866-b1ea-148689b1656d)

Business dashboard (note in URL that id=BCxxx):
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/8e0223a4-eb87-4e18-ab5c-2519ced45e99)